### ISSUE TITLE
How to merge old extended keymap to new system to make use of nkro [resolved]

### ISSUE THREAD

**Issue by mrlinuxfish** [qmk/qmk_firmware#947]
I have a few files which I cannot find in the qmk firmware and I am unsure where to make modifications so I can update my keyboard firmware. I hand wired a plank a few years ago and I really like the keyboard, but I would like to get nkro to work on the keyboard (at the time the lufa firmware did not support nkro). I need to know where to modify the matrix.c file as I'm not using a standard pinout on the teensy I used. Also I cannot find the led.c file where the pins for leds are defined (I use leds for denoting caps lock and layer indicators). Lastly, is there anything necessary to modify an extended keymap for the new qmk keymap format?

Thanks
-Dustin

**Comment by mrlinuxfish** [comment]
I found the option to define the matrix in the config.h file and I'm currently working to use make to test the firmware, but the output of my terminal keeps outputting 
`Making planck/rev4 with keymap dustin`
and it does not appear to be building the firmware. I have also tried compiling the default firmware (thinking if anything it would be functional) and it is exhibiting the same behavior.

**Comment by mrlinuxfish** [comment]
I think I found where I can put the code which used to be in led.c, but the firmware still won't compile for me. I left it trying to compile for about an hour and then when I tried to send an interrupt so I could close the terminal and try other things, windows crashed on me. I'm not sure what the issue is, but the old version of the firmware would compile correctly with exactly the same setup on my computer.
